app/default/props.conf

```
@@ -25,6 +25,8 @@ description = Messages about actions taken by the Heroku platform.
 
 EXTRACT-heroku_component,dyno = ^\S+\s(?<component>\S+)\s(?<dyno>\S+)
 EXTRACT-message = \s-\s(?<message>.*)
+EXTRACT-state = State changed from \w+ to (?<State>\w+)$
+EXTRACT-exit_code = Process exited with status (?<Exit_Code>\w+)$
```

Nailed it!

I'd lowercase the field names to be consistent with our other field names, but this looks brillant!

```suggestion
EXTRACT-state = State changed from \w+ to (?<state>\w+)$
EXTRACT-exit_code = Process exited with status (?<exit_code>\w+)$
```